osf_tests/factories.py

```diff
@@ -453,20 +453,20 @@ def add_approval_step(reg):
         reg.save()
         return reg
 
+
 class WithdrawnRegistrationFactory(BaseNodeFactory):
 
     @classmethod
     def _create(cls, *args, **kwargs):
 
-        registration = kwargs.pop('registration', None)
+        registration = kwargs.pop('registration', RegistrationFactory())
         registration.is_public = True
         user = kwargs.pop('user', registration.creator)
 
         registration.retract_registration(user)
         withdrawal = registration.retraction
         token = list(withdrawal.approval_state.values())[0]['approval_token']
-        with patch('osf.models.AbstractNode.update_search'):
```

Comment:
Minor: why was this context removed?

Reply:
It had no effect just superfluous mocking I removed for debugging, it didn't effect the test so I left it out.

Reply:
Seemingly, it prevented a potentially-unnecessary call to ES, i.e. sped up runtime slightly for every call to `WithdrawnRegistrationFactory([...])`